package.json

```diff
@@ -26,11 +26,13 @@
 		"typescript": "4.7.4"
 	},
 	"dependencies": {
+		"@types/moment-timezone": "^0.5.30",
 		"@zip.js/zip.js": "2.7.20",
 		"joplin-turndown-plugin-gfm": "1.0.12",
+		"moment-timezone": "^0.5.45",
```

Comment:
Can this change be accomplished without adding this library? Obsidian already ships with moment, so we'd like to avoid adding another version if possible.

Reply:
I wasn't sure how to get a TZ-aware function in place since `moment-timezone` is separate (and requires `moment` on its own). I'd also rather not add it, but couldn't sort out how to get the Unix timestamp into the local time (based on Daylight Saving rules too). Open to options if you know of anything that would work through moment itself - that's what I started with

Reply:
Switched back to using just `moment` (and imported from `obsidian`), the timestamps showing in Roam seem to be consistent with my TZ so importing _should_ keep it correct. The more I was digging through it, the less consistent Roam was with dates - this should be more than good enough for the import, and can always be manually adjusted for outliers

Reply:
Thank you, can you remove the package-lock and package.json changes from this PR then?

Reply:
Done, cleaned up any references to `moment-timezone` (and the types package that went with it)